src/Http/Middleware/HandleErrors.php

```diff
@@ -52,7 +52,7 @@ public function __invoke($error, Request $request, Response $response, callable
             $status = $errorCode;
         }
 
-        if ($this->debug && $errorCode !== 404) {
+        if ($this->debug && ! in_array($errorCode, [403, 404])) {
```

Comment:
If need be, we can include a (pretty-ish) stack traces on any error pages where it might make sense, if debug mode is enabled.